marked-element.html

```diff
@@ -75,7 +75,7 @@
         display: none !important;
```

Comment:
It can be replaced with a `hidden` attribute, but the style will still have to stick around since any `display` style on :host will override it